Chargebee/Classes/Purchase/CBPurchaseManager.swift

```diff
@@ -103,10 +103,12 @@ public extension CBPurchase {
     }
 
     //Buy the product
-    func purchaseProduct(product: CBProduct, customerId : String? = "" ,completion handler: @escaping ((_ result: Result<(status:Bool, subscriptionId:String?, planId:String?), Error>) -> Void)) {
+    func purchaseProduct(product: CBProduct, userInfo : CBCustomer? = nil, completion handler: @escaping ((_ result: Result<(status:Bool, subscriptionId:String?, planId:String?), Error>) -> Void)) {
```

Comment:
This modifies an existing public method. Instead, can we introduce a new method and mark this as deprecated?

Reply:
updated